Brokerages/Alpaca/AlpacaBrokerage.Utility.cs

```diff
@@ -146,6 +146,13 @@ private void OnTradeUpdate(ITradeUpdate trade)
                         FillQuantity = fillQuantity * (order.Direction == OrderDirection.Buy ? 1 : -1)
                     });
                 }
+                else if (trade.Event == TradeEvent.Rejected)
```

Comment:
Nice 👍 
They also have a `TradeEvent.Expired` which we should probably add too?

Reply:
Good idea 👍 
I think `TradeEvent.Expired` should be mapped to `OrderStatus.Canceled`